src/Common/test-runtime/LatestDependencies/project.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
I'm not sure why that would prevent it from working--the copied and modified project.json should be in `bin` where the validator doesn't look. I'm missing the context for the pull request though, so maybe I don't understand the scenario.